tests/test_simpleButler.py

```diff
@@ -356,6 +356,45 @@ def testGetCalibration(self):
                                   collections="calibs", instrument="Cam1")
         self.assertEqual(bias3b_id, bias3b.id)
 
+    def testRegistryDefaults(self):
+        """Test that we can default the collections and some data ID keys when
+        constructing a butler.
+
+        Many tests that use default run already exist in ``test_butler.py``, so
+        that isn't tested here.  And while most of this functionality is
+        implemented in `Registry`, we test it here instead of
+        ``daf/butler/tests/registry.py`` because it shouldn't depend on the
+        database backend at all.
+        """
+        butler = self.makeButler(writeable=True)
+        butler.import_(filename=os.path.join(TESTDIR, "data", "registry", "base.yaml"))
+        butler.import_(filename=os.path.join(TESTDIR, "data", "registry", "datasets.yaml"))
+        # Need to actually set defaults later, not at construction, because
```

Comment:
Can you though test that the kwargs work with `Butler` construction? Maybe by creating new butler from this butler with different defaults.

Is there a test where you have two instruments registered and so defaulting fails?

Is there a test where you specify an explicit instrument default? The tests here seem to be using the implicit "we only have one instrument in this collection so use that" code path.

Reply:
More tests added.